utils/setter.go

```diff
@@ -0,0 +1,43 @@
+package utils
+
+import (
+	"fmt"
+	"reflect"
+	"strconv"
+	"strings"
+)
+
+// SetValue assign value to the field
+// name of the struct dest.
+// Returns an error if name is not found or dest cannot
+// be addressed.
+func SetValue(name string, dest interface{}, value string) error {
+	tmp := reflect.Indirect(reflect.ValueOf(dest)).FieldByNameFunc(func(s string) bool {
+		return strings.ToUpper(s) == strings.ToUpper(name)
```

Comment:
Why do you need to do a case insensitive search?

In any case, this could be strings.EqualFold.

Reply:
This is because protoc doesn't follow the initialisms in their namings. So in `options.pb.go`, you'll find `TokenUrl` and in gunk you'll find `TokenURL`.

Reply:
I see - could you add this as a comment in the code?